web/app/css/styles.css

```diff
@@ -52,3 +52,18 @@ div.success-rate-dot {
     margin-bottom: 12px;
   }
 }
+
+div.octopus-graph-container {
+  overflow-x: scroll;
+}
+
+div.octopus-graph {
+  max-width: 1126px;
+  min-width:1126px;
```

Comment:
not sure if this is the cause, but on laptop-size screen, getting weird sizing / overflow issues:

<img width="1272" alt="screen shot 2018-10-31 at 6 57 15 pm" src="https://user-images.githubusercontent.com/236915/47828136-f7ce3f80-dd3e-11e8-8987-49a0744277d4.png">

<img width="1257" alt="screen shot 2018-10-31 at 6 56 55 pm" src="https://user-images.githubusercontent.com/236915/47828139-f9980300-dd3e-11e8-9faa-5d939584f4ee.png">

Reply:
thanks @siggy ! turns out I'd been viewing the dashboard at 90% zoom in chrome all along. I've reduced the sizes, should be good now!